### ISSUE TITLE
Stories that don't work

### ISSUE THREAD

**Issue by Krzysztof23** [bttv-android/bttv#675]
### Existing Issues

- [X] I have searched for existing issues, this is not a duplicate.

### Unmodded Twitch

- [X] I have checked and this issue only occurs using bttv-android and not with the unmodded Twitch App

### What happened?

after updating your application to the latest version, I have noticed 2-3 errors so far 
1. Streamers' stories do not work 
2. I cannot tag people in the chat in the new version of the chat/chat as an overlay on the stream screen
3. guess (to be confirmed) auto claiming
 points. channel does not work when the chat is in the form of an overlay on the stream image, and when the stream is displayed in the "picture in picture" function, a small window above other applications
![Screenshot_20240823-231247](https://github.com/user-attachments/assets/33c9565c-7990-414b-bd68-4bb40e6c3290)
![Screenshot_20240824-201635](https://github.com/user-attachments/assets/4339ca04-96fc-4a76-b805-7ff9cb2361c0)

### bttv-android version

0,41,0 -beta

### android version

11

### device

Realme C25Y

**Comment by Zerwin** [comment]
I was a bit afraid of that, when I rebased to the app version that supports stories, none of the streamers I follow had stories up, so I couldn't actually test them. However, seeing as we didn't change anything about this feature and it's basically untouched by the patches, I would guess it's a bug in the base app. 

If you could, can you check if stories work with this app [here ](https://www.apkmirror.com/apk/twitch-interactive-inc/twitch/twitch-16-9-1-release/twitch-live-game-streaming-16-9-1-2-android-apk-download/) ? This is the base twitch app without any of our changes but the same version, so if stories also don't work there it's a base twitch bug that will probably be resolved when we rebase to a newer version.

**Comment by Krzysztof23** [comment]
indeed, the errors I described seem to be caused by manufacturing defects in the main application without your modifications. The stories are not displayed at all in the base application of the given compilation, so there is probably a problem there, there is also an identical bug with the text. I apologize for the de facto incorrect reporting, as the fault is not yours. When I described the problem, I referred to the newest version of the base twitch application
